client/src/components/AddNewProfile.js

```diff
@@ -54,8 +75,34 @@ const AddNewProfile = () => {
     { key: "inactive", value: "inactive", text: "Inactive" }
   ];
 
+  const genderOptions = [
+    {key: "male", value: "male", text: "Male"},
+    {key: "female", value: "female", text: "Female"},
+    {key: "other", value: "other", text: "Other"},
+    {key: "not_provided", value: "not_provided", text: "Not provided"}
+  ];
+ 
+  useEffect(() => {
+     getNationalities().then(response => {
+      console.log("res",response)
+      setNationalities(response); 
+    });
+  }, []);
+
+  const nationalityOptions = nationalities.map(nationality => ({
+    key: nationality.id,
+    text: nationality.nationality,
+    value: nationality.nationality
+  }));
+console.log(nationalities)//to remove
+console.log(profileData)//to remove
+console.log("options", nationalityOptions)
   return (
+      
     <Form onSubmit={createProfile}>
+      <ul class="errors">
+                <p>{ errors }</p>
```

Comment:
```suggestion
         { errors }
```
I don't think a `<p>` tag is allowed as a child of `<ul>`. It shouldn't be needed here in any case.